tests/acceptance/util.py

```diff
@@ -165,7 +165,6 @@ def get_patched_runner(config_path, logger):
         The patched logprep runner
     """
     runner = Runner(bypass_check_to_obtain_non_singleton_instance=True)
-    runner.set_logger(logger)
```

Comment:
I think logger can be removed from the arguments of the method, since it isn't used anymore.

Reply:
sure... removed it ... thank you